webnode/src/redux/epics/node-epics.js

```diff
@@ -11,12 +9,10 @@ import iota from "../services/iota";
 
 import Datamap from "datamap-generator";
 
-import AppUtils from "../../utils/app";
 
 import {
   MIN_GENESIS_HASHES,
   MIN_BROKER_NODES,
-  SECTOR_STATUS,
   CHUNKS_PER_SECTOR,
```

Comment:
We don't need the SECTOR_STATUS anymore?

Reply:
this was unused, caught by the linter.